src/components/CorrectionReviewBoxes/CorrectionReviewBoxes.js

```diff
@@ -3,6 +3,7 @@ import './CorrectionReviewBoxes.scss';
 import Calendar from '../../assets/icons/calendar.svg';
 import User from '../../assets/icons/user.svg';
 import Edit from '../../assets/icons/edit.svg';
```

Comment:
Wäre es nicht möglich die Icons als Komponente zu verwenden

Reply:
die gibt es schon, die werden aber in einem extra MR eingebaut